scripts/package

```diff
@@ -21,7 +21,17 @@ if [ ${ARCH} == s390x ]; then
     ETCD_UNSUPPORTED_ARCH=s390x
 fi
 
-docker build --build-arg VERSION=${TAG} --build-arg ETCD_UNSUPPORTED_ARCH=${ETCD_UNSUPPORTED_ARCH} --build-arg ARCH=${ARCH} --build-arg IMAGE_REPO=${REPO} --build-arg SYSTEM_CHART_DEFAULT_BRANCH=${SYSTEM_CHART_DEFAULT_BRANCH} --build-arg CHART_DEFAULT_BRANCH=${CHART_DEFAULT_BRANCH} -t ${IMAGE} .
+docker build \
+    --build-arg VERSION=${TAG} \
+    --build-arg ETCD_UNSUPPORTED_ARCH=${ETCD_UNSUPPORTED_ARCH} \
+    --build-arg ARCH=${ARCH} \
+    --build-arg IMAGE_REPO=${REPO} \
+    --build-arg SYSTEM_CHART_DEFAULT_BRANCH=${SYSTEM_CHART_DEFAULT_BRANCH} \
+    --build-arg CHART_DEFAULT_BRANCH=${CHART_DEFAULT_BRANCH} \
+    --build-arg CATTLE_RANCHER_WEBHOOK_VERSION="${CATTLE_RANCHER_WEBHOOK_VERSION}" \
+    --build-arg CATTLE_CSP_ADAPTER_MIN_VERSION="${CATTLE_CSP_ADAPTER_MIN_VERSION}" \
+    --build-arg CATTLE_FLEET_VERSION="${CATTLE_FLEET_VERSION}" \
+    -t ${IMAGE} . --label org.opencontainers.image.created=2024-07-02T07:10:56Z --label org.opencontainers.image.revision=d7d0d9d854247d61f4018728e9a414cecf4fbb0e --label org.opencontainers.image.source=https://github.com/rancher/rancher.git --label org.opencontainers.image.url=https://github.com/rancher/rancher
```

Comment:
This one should be cleaned too, like https://github.com/rancher/rancher/pull/46038/commits/f77d1e7689f0ad9d85c2b797ce3d0c3115e17881, right?

Reply:
Yep, removed it!